list

```diff
@@ -803,6 +803,7 @@ migre.me
 minify.link
 minilink.io
 mklnd.com
+mnge.co
```

Comment:
Believe we could block the top-level domain, as it does not load a webpage too dissimilar from its subdomains?

Reply:
Actually I don't see it as a block list, but I think it should be neutral, and I'd prefer to use exact match, because we don't know if we can or we should assume the downstream clients will all support it.

Reply:
Ah! I saw this as a blocklist.

I'll close this PR. Please feel free to add the other subdomains as you see fit (: